bin/zfind_repeats

```diff
@@ -88,12 +88,12 @@ mjds = []
 fibers = []
 
 if set == 1:
-    for object_id in object_ids.keys():
+    for object_id in list(object_ids.keys()):
```

Comment:
This one is a bit odd, because if you look further down, there's also `for object_id in list(object_ids.values()):`.